cypress/e2e/shared/checks.test.ts

```diff
@@ -537,8 +545,96 @@ describe('Checks', () => {
         // Need to trigger mouseout else the popover obscures the other buttons
         .trigger('mouseout')
 
+      // implicitly verify id value while setting up intercept
+      cy.getByTestID('copy-resource-id').then(elem => {
+        const checkID = elem
+          .text()
+          .replace(/^\D+/g, '')
+          .substring(0, 16)
+        return cy
+          .intercept('PATCH', `/api/v2/checks/${checkID}*`)
+          .as('patchCheck')
+      })
+
+      // edit the name
+      cy.getByTestID('check-card--name-button').click()
+      cy.getByTestID('check-card--input')
+        .clear()
+        .type(newName + '{enter}')
+
+      cy.wait('@patchCheck').then(({response}) => {
+        if (response) {
+          expect(response.body.name).to.equal(newName)
+        } else {
+          fail('no response from patch name call')
+        }
+      })
+
+      // verify last completed and last updated
+      cy.getByTestID('resource-list--meta').within(() => {
+        cy.contains('Last completed').then((elem: any) => {
+          const now = new Date()
+          const text = elem.text()
+          const cdate = new Date(text.replace(/^\D+/g, ''))
+          // assert last completed value timestamp is sane
+          expect(now.getTime() - cdate.getTime())
+            .to.be.at.least(0)
+            .and.to.be.below(Cypress.config('requestTimeout'))
+        })
+        cy.contains('Last updated').then((elem: any) => {
+          const interval = parseInt(elem.text().replace(/\D+/g, ''))
+          // assert last update value is sane
+          expect(interval)
+            .to.be.at.least(0)
+            .and.to.be.at.most(Cypress.config('requestTimeout') / 1000)
```

Comment:
I noticed here you're dividing by 1000 but on line 582 you're not. Is that intentional?

Reply:
Yes.  On line 582 the comparison is of the value of the last completed time in date format, which is converted to a date object, from which the time in milliseconds is retrieved and compared to now.
```javascript
          expect(now.getTime() - cdate.getTime())
            .to.be.at.least(0)
            .and.to.be.below(Cypress.config('requestTimeout'))
```
While here the value is from the field 'Last updated _n_ seconds ago', so the value is already the difference parsed to int and represents seconds.

Reply:
got it, cool!